Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -808,14 +809,35 @@ void ConsoleVariableItemDelegate::setModelData(QWidget* editor, QAbstractItemMod
 {
     if (auto* doubleEditor = qobject_cast<AzQtComponents::SliderDoubleCombo*>(editor))
     {
+        QRegExp rx("((-?\\d+)(\\.\\d+)?)");
+        if (!rx.exactMatch(QString::number(doubleEditor->value())))
```

Comment:
A decimal point is locale specific.
In some locales it is `<comma>` instead of a `<dot>`